_drafts/2023-03-01-announcing-call-for-proposals.md

```diff
@@ -1,25 +1,30 @@
 ---
 author: DjangoCon US Organizers
 category: General
-date: 2022-05-10 06:00:00
-image: /static/img/blog/cfp-announced-2022.png
+date: 2023-03-01 06:00:00
+# TODO: Need image
+# image: 
 layout: post
-post_photo_alt: Speaker addressing a crowd at DjangoCon US 2019 in San Diego
-post_photo_url: /static/img/blog/speaking-2022-cfp.jpg
-title: The Call for Proposals for DjangoCon US 2022 Is Now Open!
+# TODO: Need image and alt
+# post_photo_alt:
+# post_photo_url:
+title: The Call for Proposals for DjangoCon US 2023 Is Now Open!
 ---
 
-The DjangoCon 2022 organizers are excited to announce that the first in-person DjangoCon since 2019 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-2022/cfp)! The deadline for submissions is June 10th, 2022 [AoE](https://time.is/compare/0000_10_June_2022_in_Anywhere_on_Earth). As long as it’s still June 10th anywhere on earth, you can submit your proposal.
+We are excited that DjangoCon US 2023 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-2023/cfp)! The deadline for submissions is May 15th, 2023 at [6 PM EDT](https://time.is/0600PM_15_May_2023_in_Durham,_United_States?DjangoCon_US_CFP_Closes).
 
-We invite you to submit your proposal no matter your background or experience level with Django. Proposals can be from a wide range of topics; non-Django and community topics are welcome. You can look at our [talk schedule](https://2021.djangocon.us/talks/) from last year for reference.
+We invite you to submit your proposal no matter your background or experience level with Django. Proposals can be from a wide range of topics; non-Django and community topics are welcome. You can look at our [talk schedule](https://2023.djangocon.us/talks/) from last year for reference.
```

Comment:
```suggestion
We invite you to submit your proposal no matter your background or experience level with Django. Proposals can be from a wide range of topics; non-Django and community topics are welcome. You can look at our [talk schedule](https://2022.djangocon.us/talks/) from last year for reference.
```

We don't have talks this year yet, so the link to last year needs to point to 2022.